### ISSUE TITLE
warnings in logs "The color-adjust shorthand is currently deprecated"

### ISSUE THREAD

**Issue by acidjazz** [windicss/nuxt-windicss#174]
```
Nuxt CLI v3.0.0-rc.1                                                                                                    17:35:29
                                                                                                                        17:35:29
  > Local:    http://localhost:3000/
  > Network:  http://172.30.42.209:3000/

ℹ nuxt-windicss v2.3.1 running with config: ./windi.config.ts.                                            nuxt-windicss 17:35:31

 WARN  [vite:css] Replace color-adjust to print-color-adjust. The color-adjust shorthand is currently deprecated.       17:35:32
31 |    padding: 0;
32 |    -webkit-print-color-adjust: exact;
33 |    color-adjust: exact;
   |     ^
34 |    display: inline-block;
35 |    vertical-align: middle;
```

**Comment by bizliaoyuan** [comment]
> dev
> nuxt dev
Nuxt CLI v3.0.0-rc.2                                                                                                      00:11:11
                                                                                                                          00:11:11
  > Local:    http://localhost:3000/
  > Network:  http://192.168.31.145:3000/

i nuxt-windicss v2.3.1 running with config: inline.                                                         nuxt-windicss 00:11:15

 WARN  [vite:css] Replace color-adjust to print-color-adjust. The color-adjust shorthand is currently deprecated.         00:11:18
31 |    padding: 0;
32 |    -webkit-print-color-adjust: exact;
33 |    color-adjust: exact;
   |     ^
34 |    display: inline-block;
35 |    vertical-align: middle;


 WARN  [vite:css] Replace color-adjust to print-color-adjust. The color-adjust shorthand is currently deprecated.         00:11:18
130|    padding-right: 0.75rem;
131|    -webkit-print-color-adjust: unset;
132|    color-adjust: unset;
   |     ^
133|  }
134|  [type='checkbox'] {


 WARN  [vite:css] Replace color-adjust to print-color-adjust. The color-adjust shorthand is currently deprecated.         00:11:18
267|    padding-right: 2.5rem;
268|    -webkit-print-color-adjust: exact;
269|    color-adjust: exact;
   |     ^
270|  }
271|  ul {

i Vite client warmed up in 1722ms                                                                                         00:11:18
i Vite server warmed up in 280ms                                                                                          00:11:18

 WARN  [vite:css] Replace color-adjust to print-color-adjust. The color-adjust shorthand is currently deprecated.         00:11:19
31 |    padding: 0;
32 |    -webkit-print-color-adjust: exact;
33 |    color-adjust: exact;
   |     ^
34 |    display: inline-block;
35 |    vertical-align: middle;


 WARN  [vite:css] Replace color-adjust to print-color-adjust. The color-adjust shorthand is currently deprecated.         00:11:19
130|    padding-right: 0.75rem;
131|    -webkit-print-color-adjust: unset;
132|    color-adjust: unset;
   |     ^
133|  }
134|  [type='checkbox'] {


 WARN  [vite:css] Replace color-adjust to print-color-adjust. The color-adjust shorthand is currently deprecated.         00:11:19
267|    padding-right: 2.5rem;
268|    -webkit-print-color-adjust: exact;
269|    color-adjust: exact;
   |     ^
270|  }
271|  ul {

√ Vite server built in 6930ms                                                                                             00:11:25
√ Nitro built in 775 ms                                                                                             nitro 00:11:26

 WARN  [vite:css] Replace color-adjust to print-color-adjust. The color-adjust shorthand is currently deprecated.         00:11:28
31 |    padding: 0;
32 |    -webkit-print-color-adjust: exact;
33 |    color-adjust: exact;
   |     ^
34 |    display: inline-block;
35 |    vertical-align: middle;


 WARN  [vite:css] Replace color-adjust to print-color-adjust. The color-adjust shorthand is currently deprecated.         00:11:28
130|    padding-right: 0.75rem;
131|    -webkit-print-color-adjust: unset;
132|    color-adjust: unset;
   |     ^
133|  }
134|  [type='checkbox'] {

**Comment by sophie721** [comment]
windicss/windicss#775
fixed after windicss v3.5.2
